src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let count = 0;
+
+  if (days < 3) {
+    count = days * 40 - 0;
+  }
```

Comment:
no need for this if with 0, it would be default case
create total = days * 40 and change it in the next 2 if block, then return total